mw-webapp/src/router/Router.tsx

```diff
@@ -16,6 +16,10 @@ export const Router = () => {
         path={pages.page404.path}
         element={pages.page404.element}
       />
+      <Route
+        path={pages.waysPage.path}
+        element={pages.waysPage.element}
+      />
```

Comment:
Move before 404 route

Reply:
done

Reply:
You forgot to push comments? It still after page 404